notdiamond/settings.py

```diff
@@ -19,7 +19,7 @@
 
 
 NOTDIAMOND_API_URL = os.getenv(
-    "NOTDIAMOND_API_URL", "https://not-diamond-server.onrender.com"
+    "NOTDIAMOND_API_URL", "https://api.notdiamond.ai"
```

Comment:
Did I mention making this change? I forgot whether I had told you!

Reply:
Saw it on readme